ecs/env.json

```diff
@@ -197,6 +197,22 @@
     {
       "name": "M365_EXCEL_INTERVAL_BETWEEN_ACTIONS_MS",
       "valueFrom": "plumber-<ENVIRONMENT>-m365-excel-interval-between-actions-ms"
+    },
+    {
+      "name": "POSTMAN_SMS_API_BASE_URL",
+      "valueFrom": "plumber-<ENVIRONMENT>-postman-sms-api-base-url"
```

Comment:
in future, can probably be stored with the connection like paysg

Reply:
actually i might as well change it to this approach now

Reply:
done, lmk what you think